modules/nw-network-observability-operator.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+// * networking/network_observability/understanding-network-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-network-observability-operator_{context}"]
+= Network Observability Operator
+
+When you deploy flow collector, the Operator provides dashboards, metrics, and keeps flows accessible in a queryable log store, Grafana Loki. When a flow collector is deployed, new dashboards are available in the Console.
+
+.Procedure
+
+. Run the following command to view the state of `flowcollector`:
++
+[source,terminal]
+----
+$ oc get flowcollector/cluster
+----
++
+.Example output
+----
+NAME      AGENT   SAMPLING (EBPF)   DEPLOYMENT MODEL   STATUS
+cluster   EBPF    50                DIRECT             Ready
+----
+
+. Check the status of pods running in namespace `netobserv`.
```

Comment:
Maybe...

. Check the status of pods running in the `netobserv` namespace by entering the following command:

Reply:
Addressed